selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/NewWorkspacePageTest.java

```diff
@@ -51,6 +51,9 @@
   private static final double MAX_RAM_VALUE = 100.0;
   private static final double MIN_RAM_VALUE = 0.5;
   private static final double RAM_CHANGE_STEP = 0.5;
+  private static final String JDK_SUGGESTION_TITLE = "JDK";
+  private static final String JAVA_SUGGESTION_TITLE = "JAVA";
+  private static final String JAVA_1_8_SUGGESTION_TITLE = "JAVA 1.8";
   private static final String NAME_WITH_ONE_HUNDRED_SYMBOLS =
```

Comment:
Actually, not related to your changes, but maybe will be useful for you. 
We have utility class for name generation `NameGenerator`. With it you can generate name with given prefix and length. So for generate workspace name with 100 symbols you can call : `NameGenerator.generate("wksp-", 95)`